test/unit_tests/modules/nav-primary-spec.js

```diff
@@ -45,61 +47,141 @@ describe( 'Get Event States', function() {
   } );
 
   describe( 'Primary Nav Link Events - Default state', function() {
+    beforeEach( function() {
+      $link = $( $( '.sub-nav-1' ).find( 'a' )[0] );
 
-    it( 'should not navigate away from the page', function() {
       $( '.nav-link-1' ).trigger( 'click' );
+    } );
 
+    it( 'should not navigate away from the page', function() {
       expect( window.location.href ).to.not.equal( 'http:// www.google.com/' );
     } );
 
     it( 'should toggle the sibling sub nav', function() {
-      $( '.nav-link-1' ).trigger( 'click' );
-
       expect( $( '.sub-nav-1' ).attr( 'aria-expanded' ) ).to.equal( 'true' );
     } );
 
     it( 'should not toggle a non-sibling sub nav', function() {
-      $( '.nav-link-2' ).trigger( 'click' );
+      expect( $( '.sub-nav-2' ).attr( 'aria-expanded' ) ).to.equal( 'false' );
+    } );
 
-      expect( $( '.sub-nav-1' ).attr( 'aria-expanded' ) ).to.equal( 'false' );
+    it( 'should focus the first link in the sibling sub nav', function( done ) {
+      setTimeout( function() { // eslint-disable-line max-nested-callbacks, no-inline-comments, max-len
+        expect( $link.is( ':focus' ) ).to.be.true;
+        done();
+      }, 500 );
     } );
-  } );
 
-  describe( 'Primary Nav Link Events - One expanded', function() {
-    it( 'should close itself', function() {
-      $( '.sub-nav-1' ).attr( 'aria-expanded', 'true' );
-      $( '.nav-link-1' ).trigger( 'click' );
+    it( 'should not focus the first link in a non-sibling sub nav', function( done ) {
+      var $nextlink = $( $( '.sub-nav-2' ).find( 'a' )[0] );
 
-      expect( $( '.sub-nav-1' ).attr( 'aria-expanded' ) ).to.equal( 'false' );
+      setTimeout( function() { // eslint-disable-line max-nested-callbacks, no-inline-comments, max-len
+        expect( $nextlink.is( ':focus' ) ).to.be.false;
+        done();
+      }, 500 );
     } );
+  } );
+
+  describe( 'Primary Nav Link Events - One expanded', function() {
+    beforeEach( function() {
+      $link = $( $( '.sub-nav-2' ).find( 'a' )[0] );
 
-    it( 'should close others, then open sibling sub nav', function() {
       $( '.sub-nav-1' ).attr( 'aria-expanded', 'true' );
       $( '.nav-link-2' ).trigger( 'click' );
+    } );
 
+    it( 'should close others', function() {
       expect( $( '.sub-nav-1' ).attr( 'aria-expanded' ) ).to.equal( 'false' );
+    } );
 
+    it( 'should open the sibling sub nav', function( done ) {
       setTimeout( function() { // eslint-disable-line max-nested-callbacks, no-inline-comments, max-len
         expect( $( '.sub-nav-2' ).attr( 'aria-expanded' ) ).to.equal( 'true' );
+        done();
+      }, 500 );
+    } );
+
+    it( 'should focus the first link in sibling sub nav', function( done ) {
+      setTimeout( function() { // eslint-disable-line max-nested-callbacks, no-inline-comments, max-len
```

Comment:
Consider adding a `max-nested-callbacks` override to https://github.com/cfpb/cfgov-refresh/blob/flapjack/test/unit_tests/.eslintrc . Ignore-lines should be one-off snowflakes, but if many tests are needing this and there isn't a way to refactor this away, an override in the eslint file for the tests makes sense.